cli/new.md

```diff
@@ -19,7 +19,7 @@ Options:
   --no-color  Disable colored output
   -t          Selects the template engine language, can be one of: slang | ecr (default: slang)
   -h, --help  Show this help
-  -r          Use a named recipe.  See documentation at https://amberframework.org.
+  -r          Use a named recipe.  See documentation at https://amberframework.gitbook.io/amber/cli/recipes.
```

Comment:
There is another place to change this: https://github.com/amberframework/docs/blob/d3f8076846b65ecbe6e4234331724b4aab88fc8a/cli/recipes.md

Reply:
Fix this:
https://github.com/amberframework/docs/pull/85/commits/9821628cadc113a070e7578c2a6d0a7b46ed7eb1